datasets/hebrew_this_world/hebrew_this_world.py

```diff
@@ -0,0 +1,95 @@
+"""HebrewThisWorld: A corpus from https://thisworld.online/."""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import ctypes
+
+import datasets
+
+
+_DESCRIPTION = """\
+HebrewThisWorld is a data set consists of 2028 issues of the newspaper 'This World' edited by Uri Avnery and were published between 1950 and 1989. Released under the AGPLv3 license."""
+
+csv.field_size_limit(int(ctypes.c_ulong(-1).value // 2))
```

Comment:
What does this do ? Is it necessary ?

Reply:
Hi, you mean extending csv dict size? Unfortunately yes, tried firstly without it but the text content is more than limit of the CSV reader - https://stackoverflow.com/questions/15063936/csv-error-field-larger-than-field-limit-131072
Then I found this line in the other dataset at the project